project/plugins.sbt

```diff
@@ -0,0 +1,10 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.4")
+
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "0.9.8")
+
+resolvers += Resolver.url(
+  "idio",
+  url("http://dl.bintray.com/idio/sbt-plugins")
+)(Resolver.ivyStylePatterns)
+
+addSbtPlugin("org.idio" % "sbt-assembly-log4j2" % "0.1.0")
```

Comment:
this is needed as it seems presto and lambda-log4j both pull in a dat file which needs a special merge.